### ISSUE TITLE
Missing features with traditional titlebar

### ISSUE THREAD

**Issue by Idesmi** [johnfactotum/foliate#343]
There isn't a way to change theme, font, spacing, margins while using the (experimental) traditional titlebar.
This is what's missing:
![This](https://user-images.githubusercontent.com/21034950/81536773-9b000c00-935b-11ea-9c76-9b3d6d91fa05.png)

**Comment by johnfactotum** [comment]
Yeah, that's really the main reason why it's marked as experimental. Well, this and also not being able to view table of contents/bookmarks/annotations.

**Comment by bugzy** [comment]
Just to put in a proper list of missing features in this mode and workarounds that are available:
- [ ] No way to search (CTRL + F or / do not work)
- [ ] No way to adjust themes, fonts, spacing, margin
- [ ] No way to view library or catalogs
- [ ] No way to view "table of contents/bookmarks/annotations" (Workaround: Edit > Preferences> Use sidebar )


As a side note, I have always used foliate with the traditional title bar so I did not realize until a year ago that there was a library or catalogs that I could download books from (I was generally confused about the related setting in preferences)

**Comment by dxps** [comment]
And something that I'd see useful to have in "traditional titlebar" mode is to be able to hide the menu, with the existing popular options to show with using Alt (or F10, for example).

**Comment by johnfactotum** [comment]
The GTK 4 version doesn't have a traditional titlebar mode anymore (and probably won't get one any time soon) so I'm morphing this issue into "add traditional titlebar mode".
